packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx

```diff
@@ -417,262 +429,274 @@ export const CreateCluster = () => {
-                }}
-                handleIPv6Change={(newIpV6Addr: ExtendedIP[]) => {
-                  const validatedIPs = validateIPs(newIpV6Addr, {
-                    allowEmptyAddress: true,
-                    errorMessage: 'Must be a valid IPv6 address.',
-                  });
-                  setIPv6Addr(validatedIPs);
-                }}
-                handleIsAcknowledgementChecked={(isChecked: boolean) => {
-                  setIsACLAcknowledgementChecked(isChecked);
-                  setIPv4Addr([stringToExtendedIP('')]);
-                  setIPv6Addr([stringToExtendedIP('')]);
-                }}
-                ipV4Addr={ipV4Addr}
-                ipV6Addr={ipV6Addr}
-                isAcknowledgementChecked={isACLAcknowledgementChecked}
+
+              <Divider sx={{ marginBottom: 4 }} />
+              <NodePoolPanel
+                apiError={errorMap.node_pools}
+                handleConfigurePool={handleOpenNodePoolConfigDrawer}
+                hasSelectedRegion={hasSelectedRegion}
+                isAPLEnabled={aplEnabled}
+                isPlanPanelDisabled={isPlanPanelDisabled}
+                isSelectedRegionEligibleForPlan={
+                  isSelectedRegionEligibleForPlan
+                }
+                regionsData={regionsData}
+                selectedRegionId={selectedRegion?.id}
                 selectedTier={selectedTier}
-                setControlPlaneACL={setControlPlaneACL}
+                types={typesData || []}
+                typesError={
+                  typesError
+                    ? getAPIErrorOrDefault(
+                        typesError,
+                        'Error loading Linode type information.'
+                      )[0].reason
+                    : undefined
+                }
+                typesLoading={typesLoading}
               />
-            </>
-
-            <Divider sx={{ marginBottom: 4 }} />
-            <NodePoolPanel
-              apiError={errorMap.node_pools}
-              handleConfigurePool={handleOpenNodePoolConfigDrawer}
-              hasSelectedRegion={hasSelectedRegion}
-              isAPLEnabled={aplEnabled}
-              isPlanPanelDisabled={isPlanPanelDisabled}
-              isSelectedRegionEligibleForPlan={isSelectedRegionEligibleForPlan}
-              regionsData={regionsData}
-              selectedRegionId={selectedRegion?.id}
-              selectedTier={selectedTier}
-              types={typesData || []}
-              typesError={
-                typesError
-                  ? getAPIErrorOrDefault(
-                      typesError,
-                      'Error loading Linode type information.'
-                    )[0].reason
+            </Paper>
+          </Grid>
+          <Grid
+            data-testid="kube-checkout-bar"
+            size={{ lg: 3, md: 12, sm: 12, xs: 12 }}
+          >
+            <KubeCheckoutBar
+              createCluster={() => createCluster(form.getValues())}
```

Comment:
I think we can use react hook form's submit handler

```suggestion
              createCluster={form.handleSubmit(createCluster)}
```